logger.py

```diff
@@ -0,0 +1,22 @@
+# a utility class for logging
+import datetime
+import os
+import time
+
+LOG_DIRECTORY = 'logs'
+
+# Log format:
+#   [timestamp]\t[uuid]\t[source_entity]\t[destination_entity]\t[request_type]\t[status]\t[request_size]\t[response_size]
+def log(uuid, source_entity, destination_entity, request_type, status, response_size):
+    log_file = datetime.datetime.utcnow().strftime('%Y-%m-%d') + '.log'
+    timestamp = time.time()
+    log_entry = str(int(timestamp)) + '\t' + \
+                str(uuid) + '\t' + \
+                str(source_entity) + '\t' + \
+                str(destination_entity) + '\t' + \
+                str(request_type) + '\t' + \
+                str(status) + '\t' + \
+                str(response_size) + '\n'
+    log_file_path = LOG_DIRECTORY + '/' + log_file
```

Comment:
i wonder if this will break on windows machines because the main path separator is `\`? may need to use something on `os.path`